rm-datasource/src/main/java/org/apache/seata/rm/datasource/xa/ConnectionProxyXA.java

```diff
@@ -224,6 +230,10 @@ public synchronized void commit() throws SQLException {
 
     @Override
     public void rollback() throws SQLException {
+        rollback(true);
```

Comment:
我建议定义一个成员变量来替代这个方法入参,并且这变量要在close时进行重置,避免被连接池复用时参数错误
I suggest defining a member variable to replace the imported parameter of this method, and this variable should be reset at close to avoid parameter errors when reused by the connection pool

Reply:
有道理。已经修改了，加了一个 `xaEnd()` 的方法来封装 XA End 的调用